app/validation/validators.py

```diff
@@ -235,3 +232,52 @@ def __call__(self, form, from_field, to_field):
 
         if from_date == to_date or from_date > to_date:
             raise validators.ValidationError(self.messages['INVALID_DATE_RANGE'])
+
+
+class SumCheck(object):
+    def __init__(self, messages=None, currency=None):
+        if not messages:
+            messages = error_messages
+        self.messages = messages
+        self.currency = currency
+
+    def __call__(self, form, conditions, total, target_total):
+        message_tmpl = 'TOTAL_SUM_NOT_{}'
```

Comment:
I think we should get rid of this and be explicit about the error message we return in `_is_valid`
so instead of returning `EQUALS` we should just return `TOTAL_SUM_NOT_EQUALS` as there is no need to have to stitch this together